app/assets/scss/views/_service_updates.scss

```diff
@@ -0,0 +1,19 @@
+.govuk-option-select {
```

Comment:
I've made a pull request which adds this text field into the DM Toolkit (see https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/163).

I'll raise another pull request to make this app use that version once merged.